portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,115 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+ul.topnav {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #731143;
 }
+ul.topnav li {float: left;}
+
+ul.topnav li a {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+ul.topnav li a:hover {background-color: #560D32;}
```

Comment:
Use lower case for colors: #560d32
(from the style guide: https://google.github.io/styleguide/htmlcssguide.html#Capitalization)